<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGeph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="7">
     <widget class="QLineEdit" name="socksEdit">
      <property name="toolTip">
       <string>Leave blank to use default: 127.0.0.1:9909</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="serverSelectionBox"/>
    </item>
    <item row="1" column="7" rowspan="2">
     <widget class="QLabel" name="subscriptionInfo">
      <property name="toolTip">
       <string>User info</string>
      </property>
      <property name="text">
       <string>N.A.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QCheckBox" name="excludeCheckBox">
      <property name="text">
       <string>Exclude PRC traffic</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="httpEdit">
      <property name="toolTip">
       <string>Leave blank to use default: 127.0.0.1:9910</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="6">
     <widget class="QLineEdit" name="usernameEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Listening HTTP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="logFileSelectionButton">
      <property name="text">
       <string>Select file...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Log file</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>SOCKS5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QCheckBox" name="bridgesCheckBox">
      <property name="text">
       <string>Always use bridges</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="6">
     <widget class="QLineEdit" name="passwordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="tcpCheckBox">
      <property name="text">
       <string>TCP mode</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="6">
     <widget class="QLineEdit" name="logEdit">
      <property name="toolTip">
       <string>Leave blank to disable</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Geph executable</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="binarySelectionButton">
      <property name="text">
       <string>Select file...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="6">
     <widget class="QLineEdit" name="binaryPathEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="Toolbar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Toolbar</class>
   <extends>QToolBar</extends>
   <header>src/toolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
